registry/assets_for_api_docs.test

```diff
@@ -0,0 +1,7 @@
+# Tests from https://github.com/flutter/assets-for-api-docs
+
+contact=katelovett@google.com
+fetch=git -c core.longPaths=true clone https://github.com/flutter/assets-for-api-docs.git tests
+fetch=git -c core.longPaths=true -C tests checkout 109051af07cb2927f54512b58d6b25d4f515765b
+update=.
+test=./ci/check --no-activate analyze
```

Comment:
@gspencergoog am I holding this right? 🙃 

Also, did you wanted to be listed as a contact?

Reply:
Yes, that should work.  And sure, I'd be happy to be a contact.

Reply:
Although if you don't activate, then you have to make sure that the plugin tools are activated somewhere else.

Reply:
Oh! whoops, thank you!